src/service_name.json

```diff
@@ -0,0 +1,514 @@
+[
+  {
+    "Command": "az ai-examples",
+    "Description": "Add AI powered examples to help content.",
+    "AzureServiceName": "Microsoft AI",
+    "ConversationalServiceName": "Microsoft AI",
+    "ExtensionName": "ai-examples",
+    "URL": "https://docs.microsoft.com/archive/msdn-magazine/2017/connect/artificial-intelligence-getting-started-with-microsoft-ai"
+  },
+  {
+    "Command": "az alias",
+    "Description": "Support for command aliases",
+    "AzureServiceName": "Azure CLI",
+    "ConversationalServiceName": "Azure CLI",
+    "ExtensionName": "alias",
+    "URL": "https://docs.microsoft.com/cli/azure/"
```

Comment:
Where does this URL come from? The current one is a general CLI doc link.

Reply:
We can set it reference of `az alias`